### PR TITLE
Add tizen module

### PR DESCRIPTION
**Pull request by haesik**
From now, we will put tizen appfw related functions in the tizen module

IoT.js-DCO-1.0-Signed-off-by: Haesik Jun haesik.jun@samsung.com

### CODE REVIEW

File: src/modules/iotjs_module_tizen.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * limitations under the License.
+ */
+
+#ifndef IOTJS_MODULE_TIZEN_H
+#define IOTJS_MODULE_TIZEN_H
+
+void iotjs_tizen_func(const char* command, const char* message, void* handle);
+
+
+#endif /* IOTJS_MODULE_TIZEN_H */
```
Comment (hs0225) [inline src/modules/iotjs_module_tizen.h]: Is header file necessary?

File: test/profiles/tizen.profile [hunk 2]
```diff
@@ -7,3 +7,4 @@ ENABLE_MODULE_I2C
 ENABLE_MODULE_PWM
 ENABLE_MODULE_SPI
 ENABLE_MODULE_UART
+ENABLE_MODULE_TIZEN
```
Comment (hs0225) [inline test/profiles/tizen.profile]: Please sort alphabetically.

File: src/modules/iotjs_module_bridge.c [hunk 3]
```diff
@@ -157,11 +161,15 @@ static iotjs_bridge_call_t* iotjs_bridge_call_init(
     iotjs_string_t command, iotjs_string_t message) {
   if (bridge) {
     bridgecall->jobject = jerry_acquire_value(bridge);
+  } else {
+    bridgecall->jobject = 0;
```
Comment (zherczeg) [inline src/modules/iotjs_module_bridge.c]: 0 is not exactly a jerry value. What about using undefined?

File: src/modules/iotjs_module_bridge.c [hunk 4]
```diff
@@ -157,11 +161,15 @@ static iotjs_bridge_call_t* iotjs_bridge_call_init(
...
   }
   if (jcallback) {
     bridgecall->jcallback = jerry_acquire_value(jcallback);
     bridgecall->req.data = (void*)bridgecall;
     uv_mutex_init(&bridgecall->call_lock);
+  } else {
+    bridgecall->jcallback = 0;
```
Comment (zherczeg) [inline src/modules/iotjs_module_bridge.c]: Ditto.

**Pull request merged**